### PR TITLE
Improve Translation and updates :D

### PR DESCRIPTION
**Pull request by emirjorge**

### CODE REVIEW

File: web/translation/translate.es_ES.toml [hunk 1]
```diff
@@ -147,8 +147,8 @@
 "totalFlow" = "Flujo Total"
 "leaveBlankToNeverExpire" = "Dejar en Blanco para Nunca Expirar"
 "noRecommendKeepDefault" = "No hay requisitos especiales para mantener la configuración predeterminada"
-"certificatePath" = "Camino"
-"certificateContent" = "Contenido"
+"certificatePath" = "Ruta del Archivo"
+"certificateContent" = "Contenido del Archivo"
```
Comment (MHSanaei) [inline web/translation/translate.es_ES.toml]: These two are too long , please make it shorter

Comment (emirjorge) [reply web/translation/translate.es_ES.toml]: already fixed. :D

**Comment by emirjorge** [comment]
Fixed and updated, I added Spanish to Readme

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # 3X-UI
 
-[English](/README.md) | [Chinese](/README.zh.md)
+[English](/README.md) | [Chinese](/README.zh.md) | [Vietnamese](/README-VIE.md) | [Español](/README.es_ES.md)
```
Comment (MHSanaei) [inline README.md]: Please remove Vietnamese here, it's not ready yet

**Comment by MHSanaei** [comment]
Thank you

**Pull request merged**